inspirehep/modules/records/mappings/v7/records/experiments.json

```diff
@@ -194,7 +205,24 @@
         "type": "text"
       },
       "name_variants": {
-        "type": "text"
+        "type": "text",
+        "copy_to": ["normalized_name_variants_keyword", "normalized_name_variants_text"]
```

Comment:
* If you just need a different type/analyzer for the same field, you can use `fields` instead on `copy_to` (which is mainly used for combining several fields into one).
* why do you need both a `keyword` and `text` variant with different analyzers/normalizers?

Reply:
* for name_variants I'm actually copying data from 2 separate fields: `name_variants` and `collaboration.value`. but for `subgroups` you are right, I can use fields :+1:  
* This is a result of the way search is done for collaborations. In short: We want to have exact match search for collaboration to be sure that we have proper collaboration (so we need keyword type), but if exact_match is not giving any results we want to also do match query (discussion we had on my first approach to this task) which will work correctly only on text type.

Reply:
I thought we decided to use only simple keyword search with normalization? I'm a bit confused, let's discuss it on Monday.

Reply:
:+1: